sdk/servicebus/service-bus/README.md

```diff
@@ -186,7 +186,7 @@ await sender.send({
 });
 ```
 
-### Receive messages from Sessions
+### Receive messages using Sessions
```

Comment:
`from` is more appropriate here.
`Receive messages from session-enabled-entities` is more accurate.